seed/challenges/01-responsive-web-design/applied-visual-design.json

```diff
@@ -331,7 +331,7 @@
       "description": [
         "To make text bold, you can use the <code>strong</code> tag. This is often used to draw attention to text and symbolize that it is important. With the <code>strong</code> tag, the browser applies the CSS of <code>font-weight: bold;</code> to the element.",
         "<hr>",
-        "Wrap a <code>strong</code> tag around the entire <code>h4</code> element and its contents."
+        "Wrap a <code>strong</code> tag around 'Stanford University' inside the <code>p</code> tag."
```

Comment:
The test message uses double quotes, but this uses single quotes. Can you change this to double quotes?